Code/GraphMol/ChemReactions/Wrap/rdChemReactions.cpp

```diff
@@ -566,6 +576,142 @@ Sample Usage:\n\
            python::return_value_policy<python::reference_existing_object>(),
            "get the agent templates")
 
+      // properties
+      .def("SetProp", RDKit::MolSetProp<RDKit::ChemicalReaction, std::string>,
```

Comment:
Seems like we should make this easier in the future, perhaps with an RDProps wrapper base class.

Reply:
Most definitely